packages/verify-contract/src/common/url.ts

```diff
@@ -43,9 +43,13 @@ const DEFAULT_SCAN_API_URLS: Map<NetworkName, string> = new Map([
     ['arbitrum', 'https://api.arbiscan.io/api'],
     ['arbitrum-goerli', 'https://api-goerli.arbiscan.io/api'],
     ['arbitrum-mainnet', 'https://api.arbiscan.io/api'],
+    ['astar', 'https://astar.blockscout.com/api'],
+    ['astar-mainnet', 'https://astar.blockscout.com/api'],
     ['avalanche', 'https://api.snowtrace.io/api'],
     ['avalanche-mainnet', 'https://api.snowtrace.io/api'],
     ['avalanche-testnet', 'https://api-testnet.snowtrace.io/api'],
+    ['aurora', 'https://explorer.mainnet.aurora.dev/api'],
+    ['aurora-mainnet', 'https://explorer.mainnet.aurora.dev/api'],
```

Comment:
Let's retain alphabetical order and have this between astar and avalance

Reply:
whoopsies, fixed